tests/effective_dps_test.cpp

```diff
@@ -518,23 +518,23 @@ static void check_knives( const std::function<Approx( const std::string & )> &ca
     SECTION( "knives" ) { // expected value 19
         CHECK( calc_expected_dps( "bio_blade_weapon" ) == 24.5 ); // much better than any other knife
         CHECK( calc_expected_dps( "knife_combat" ) == 19.0 );
-        CHECK( calc_expected_dps( "knife_trench" ) == 18.0 );
-        CHECK( calc_expected_dps( "knife_baselard" ) == 18.0 );
+        CHECK( calc_expected_dps( "knife_trench" ) == 15.19 );
+        CHECK( calc_expected_dps( "knife_baselard" ) == 15.87 );
         CHECK( calc_expected_dps( "kirpan" ) == 18.0 );
-        CHECK( calc_expected_dps( "tanto" ) == 18.0 );
-        CHECK( calc_expected_dps( "kris" ) == 18.0 );
-        CHECK( calc_expected_dps( "knife_rambo" ) == 17.0 );
-        CHECK( calc_expected_dps( "tanto_inferior" ) == 16.5 );
-        CHECK( calc_expected_dps( "bone_knife" ) == 12.0 );
+        CHECK( calc_expected_dps( "tanto" ) == 13.15 );
```

Comment:
This is a pretty big drop.

Reply:
Pretty much every knife that was listed as dealing "stab" damage had their to-hit lowered as a result of being changed to "point" instead of "line".

Reply:
I gave them "short" type instead of "hand" type, considering one of the contributors above mentioned they can match that description.  I'm feeling generous so let's give them a little length.